### ISSUE TITLE
Update Docker images to latest (1.7.5) version

### ISSUE THREAD

**Issue by x80486** [hayd/deno-docker#110]

**Comment by danopia** [comment]
Latest is now 1.8.0

@hayd would it be helpful to submit a PR that bumps all the version references?

**Comment by hayd** [comment]
Apologies for the slowness.

Looks like there is an issue with alpine for 1.8.0 that needs to be resolved first.

https://github.com/hayd/deno-docker/runs/2024710911?check_suite_focus=true
```
deno: error while loading shared libraries: libstdc++.so.6: cannot open shared object file: No such file or directory
```

**Comment by hayd** [comment]
closed by #112 (but note that alpine requires libstdc++ and prints a version warning once it's installed, I have reported this as a deno issue).

**Comment by danopia** [comment]
Thanks for jumping on this!!

I also noticed the new c++ dep when building Deno in a Debian based image. Hopefully something they address soon. Cheers
